src/Helper/ProcessHelper.php

```diff
@@ -144,6 +157,15 @@ private function printPreStart(array $cmd): float
         $cmdString = implode(' ', $cmd);
         $startTime = microtime(true);
 
+        if ($this->jsonMode) {
+            fwrite(\STDOUT, json_encode([
+                'datetime' => (new \DateTime())->format('Y-m-d H:i:s'),
```

Comment:
without json mode, we print this deployment-helper "banner". should we also add an identifier to the JSON? :thinking: